package.json

```diff
@@ -43,11 +43,11 @@
     "esvalid": "1.1.0",
     "fs-readdir-recursive": "0.1.0",
     "jshint": "2.5.11",
-    "lodash": "2.4.1",
+    "lodash": "3.0.0",
```

Comment:
Sure thing, I probably got a bit ahead of myself by fixing all the ranges. a19321ea690f05543c6aa2c09cd1bb66e972ae56